dev_tools/generate_documentation.rst

```diff
@@ -29,30 +29,24 @@ On Ubuntu (or other Debian-based distributions), the relevant commands can be cu
 
 There are two files which may require modifications, on top of the other generated files (html, json, ...)
 
-- `README.rst`: A README update for users visiting the Github page.
+- `README.rst`: A README update for users visiting the GitHub page.
 - `index.qmd`: The main page of the rendered documentation, listing and tying together all the other rendered html files.
 
-2b. Regenerate the html files
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+2b. Locally regenerating the html files (optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 This is simply done by calling the following command in the root directory of the `Tangelo-Examples` folder:
 
 .. code-block::
 
    quarto render
 
-Note that all html pages include data about the side menu used to navigate through the website. As a result, even though you may be only adding or modifying a single file, you may find that all generated files (html, json) have slightly changed as a result. 
-Please make sure to add, commit and push all modifications, and trust the process.
+Rendered files have been generated, and they can be naviguated in the `_site` folder.
```

Comment:
ok so basically do we mean here:

- this is how you can manually render the docs (maybe we should just indicate users can navigate them with their web browser or show a command line with what I assume is index.html)
- and below we are saying that once people are happy with whats generated by the code in their PR, the doc will be overwritten automatically (and that is the only way we want contributors to update these docs)

If thats right then we can change the content of this document to explicitly advise contributors

1) Update the README

2) Generate locally, browse to make sure the docs works the way it is intended to, drop a quick line to tell them how

3) Once your PR is merged, the updated documentation is automatically deployed online

Reply:
Yes it is exactly like that. The second step is optional (if only minor changes were made). Also, we trigger a deployment manually.

Reply:
Thats the part i need to clarify. Before merging the PR, we need to know that the doc work as intended, so files need to be generated and deployed somewhere (locally, or online somewhere that hopefully does not overwrite the existing doc).

What is your recommended way to proceed to make it easy for people opening / reviewing the PR that this is working before approving ?

Reply:
Locally should be the way of testing the rendeing. The instructions are the ones in `dev_tools/generate_documentation.rst` file. Should we add something to the README to state that?

Reply:
I think we're good for now